test/controllers/games_controller_test.rb

```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+class GamesControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get games_index_url
```

Comment:
With your route config I think this should a `root_path` as that is what you have setup.